Automattic/jetpack
# [Boost] Add what to do section on page showstopper errors

Fixes #22205 

#### Changes proposed in this Pull Request:

* Show suggested steps if critical CSS has failed to generate due to page errors.

#### Jetpack product discussion
pc9hqz-d1-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Add the following mu-plugin:
```
<?php
if ( ! empty( $_GET['jb-generate-critical-css'] ) ) {
    header( 'HTTP/1.0 404 Not Found' );
    die();
}
```
* Try to generate critical CSS through boost.
* You should see the request fail with possible next steps hidden under "Show error messages" fold.

https://d.pr/v/VpXGlQ
